temoa/components/costs.py

```
@@ -295,7 +299,7 @@ def fixed_or_variable_cost(
 # ============================================================================
 
 
-def PeriodCost_rule(M: 'TemoaModel', p):
+def PeriodCost_rule(M: 'TemoaModel', p: int) -> float | Any:
```

Ruff (0.14.0)

Remove quotes from type annotation

Remove quotes

(UP037)